README.md

```diff
@@ -130,3 +130,18 @@ Overly complex structures and unnecessary code can be avoided by effective refac
 * Overly-nested and long functions
 * Long parameter lists
 
+
+## Code Reviews
+### [Evaluation Feedback process](https://smartbear.com/learn/code-review/guide-to-code-review-process/)
+* #### Use checklists
+Checklists are the most effective way to eliminate frequently made errors. Code review checklists also provide team members with clear expectations for each type of review and can be helpful to track for reporting and process improvement purposes.
+
+* #### Establish a process for fixing defects found
+The best way to ensure that defects are fixed is to use a collaborative code review tool that allows reviewers to log bugs, discuss them with the author, and approve changes in the code.
+
+* #### Foster a positive code review culture
+To make code reviews successful, it is important that managers create a positive culture of collaboration and learning in peer review.
+While it´s easy to see defects as purely negative, each bug can actually be an opportunity for the team to improve code quality. Peer review also allows junior team members to learn from senior leaders and for even the most experienced programmers to break bad habits.
+
+### Code Review Checklist
+![Code Review Checklist](https://i1.wp.com/www.evoketechnologies.com/wp-content/uploads/2015/08/Code-Review-Checklist.png?w=876&ssl=1)
```

Comment:
The picture is a bit too big is there any way you could make it smaller or find an alternative

Reply:
make smaller no find alternative  yes

Reply:
If you download the image you can resize it